clients/src/main/java/org/apache/kafka/common/requests/ProduceResponse.java

```diff
@@ -359,6 +358,25 @@ public String toString() {
         }
     }
 
+    public static final class ErrorRecord {
```

Comment:
nit: I think `RecordError` is a more natural name. For example, `recordErrors` over `errorRecords`.

Reply:
ack